recipe/meta.yaml

```diff
@@ -11,19 +11,25 @@ source:
   sha256: {{ sha256 }}
   patches:
     - 0001-remove-run-install-checks.patch
+    - 0002-bypass-fmath-non-intel-cpus.patch
+    - 0003-enable-cross-compiling-scons.patch
+    - 0004-support-scons-4.patch
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win]
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ pin_compatible('numpy') }}                                  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - scons <4
+    - scons
 
   host:
     - python
-    - numpy
+    - {{ pin_compatible('numpy') }}
```

Comment:
why did you change this line?

Reply:
This is what @isuruf told me to do but maybe it isn’t necessary here? I guess I haven’t tried without the change.

Reply:
I don't mind it I just don't understand it. I thought we needed numoy with no constraints in host for the global pinning to apply properly. 

 Have you checked the output packages?

Reply:
Their lower numoy bounds should match the global pins on each platform and not the latest versions of numoy.

Reply:
This all appears to pass without the pin-compatible statements.

Reply:
@isuruf can you comment on why you thought they were needed?

Reply:
It may very well be I added this when having unrelated solving issues.